src/insights/pool.ts

```diff
@@ -12,6 +12,14 @@ export interface Pool {
   market: string | null
 }
 
+export interface PoolTokens {
+  total: number
+  denom: string
+  amountShifted: number
+  amountValue: number
+  balance: number
```

Comment:
`balance` is a string. Again, remember to run the query yourself to check the struct
https://api-insights.carbon.network/pool/liquidity?poolId=14,15,16